### ISSUE TITLE
Crash during bodyslides import

### ISSUE THREAD

**Issue by leonardovcr** [Synthesis-Collective/SynthEBD#107]
Hi Piranha, here's yet another one:

```
SynthEBD has crashed with the following error:
Index was out of range. Must be non-negative and less than the size of the collection. (Parameter 'index')
   at SynthEBD.VM_BodySlideExchange.ImportGendered(ObservableCollection`1 currentBodySlides, List`1 importedBodySlides, List`1 multiplexWarnings) in D:\a\SynthEBD\SynthEBD\SynthEBD\Classes_Aux\ViewModels\VM_BodySlideExchange.cs:line 316
   at SynthEBD.VM_BodySlideExchange.Import() in D:\a\SynthEBD\SynthEBD\SynthEBD\Classes_Aux\ViewModels\VM_BodySlideExchange.cs:line 270
   at SynthEBD.VM_BodySlideExchange.<>c__DisplayClass1_0.<.ctor>b__1(Object _) in D:\a\SynthEBD\SynthEBD\SynthEBD\Classes_Aux\ViewModels\VM_BodySlideExchange.cs:line 33
   at SynthEBD.RelayCommand.Execute(Object parameter) in D:\a\SynthEBD\SynthEBD\SynthEBD\GUI_Aux\RelayCommand.cs:line 29
   at MS.Internal.Commands.CommandHelpers.CriticalExecuteCommandSource(ICommandSource commandSource, Boolean userInitiated)
   at System.Windows.Controls.Primitives.ButtonBase.OnClick()
   at System.Windows.Controls.Button.OnClick()
   at System.Windows.Controls.Primitives.ButtonBase.OnMouseLeftButtonUp(MouseButtonEventArgs e)
   at System.Windows.UIElement.OnMouseLeftButtonUpThunk(Object sender, MouseButtonEventArgs e)
   at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   at System.Windows.UIElement.ReRaiseEventAs(DependencyObject sender, RoutedEventArgs args, RoutedEvent newEvent)
   at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
   at System.Windows.UIElement.RaiseTrustedEvent(RoutedEventArgs args)
   at System.Windows.Input.InputManager.ProcessStagingArea()
   at System.Windows.Input.InputProviderSite.ReportInput(InputReport inputReport)
   at System.Windows.Interop.HwndMouseInputProvider.ReportInput(IntPtr hwnd, InputMode mode, Int32 timestamp, RawMouseActions actions, Int32 x, Int32 y, Int32 wheel)
   at System.Windows.Interop.HwndMouseInputProvider.FilterMessage(IntPtr hwnd, WindowMessage msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at System.Windows.Interop.HwndSource.InputFilterMessage(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)



SynthEBD Version: 0.9.7.1

Run Mode: Standalone

Patcher Settings Creation Log:
Did not find settings source path at C:\Modding\MO2\SynthEBD_v0.9.7.1-vanilla\Settings\SettingsSource.json
Using default environment and patcher settings locations.


Patcher Environment Creation Log:
Did not find settings source path at C:\Modding\MO2\SynthEBD_v0.9.7.1-vanilla\Settings\EnvironmentSource.json
Using default environment and patcher settings locations.


Patcher State:
Main Settings:
Validation Disabled: False
Apply Assets: True
Apply BodyShapes via: None
Apply Height: True
Apply Head Parts: True
Use Consistency: True
Mod Manager Type: None
Primary Config Files
MixIn Config Files
Male BodyGen Configs
Female BodyGen Configs
Male BodySlides: 39 (8) annotated
Female BodySlides: 869 (11) annotated
Height Configs: 0
Eyebrows: 0 HeadParts
Eyes: 0 HeadParts
Face: 0 HeadParts
FacialHair: 0 HeadParts
Hair: 0 HeadParts
Misc: 0 HeadParts
Scars: 0 HeadParts


Current NPC Info: Null
```

This is a brand new 0.9.7.1 installation with nothing installed aside from MO2's enabled bodyslides. Crash happens while importing an exported bodyslide .json (https://www.nexusmods.com/skyrimspecialedition/mods/73717?tab=files&file_id=362676).

**Comment by Piranha91** [comment]
I believe this should be fixed in 0.9.7.2.

**Comment by leonardovcr** [comment]
Fixed, thanks again!
